Open_Science_New.html

```diff
@@ -69,93 +65,136 @@ <h4 style="padding-bottom:0px; margin-bottom:0px; text-shadow:-1px -1px 0 #000,
+          </p>
+        </section>
+        </section>
+
+          <!-- Open Science Initiatives - Column 2 -->
+    <section data-transition="convex">
 	    <h2>Open Science Initiatives</h2>
 	    <table class="reveal">
 	      <tr>
 		<td width="47%" style="vertical-align: middle">
-		  <img src="images/Nature_open_science.png"> 
+		<a href="https://www.nature.com/articles/533S71a"><img src="images/Nature_open_science.png"></a>
 		</td>
 		<td width="53%" style="vertical-align: middle">
-		  <img src="images/Open_science.png"> 
-	    </td></tr></table>	
-	  </section>
+		<a href="https://www.sciencemag.org/news/2016/01/montreal-institute-going-open-accelerate-science"><img src="images/Open_science.png"></a>
+	    </td></tr></table>
+        <p><small>In the following columns, we'll look at some current initiatives.</small></p>
+	  </section>
+
+          <!-- MNI Initiatives - Column 3 -->
+          <section data-transition="fade">
+             <section>
+          <h1>MNI</h1>
+                 <p>Montreal Neurological Institute-Hospital</p>
+              <a href="http://www.mcgill.ca/neuro/"><img src="images/MNI-logo.png"></a>
+              <p><small>"The Neuro" integrates patient care with Open Science research.</small></p>
+                 <p><small><font color="E83812">Scroll down to explore some of MNI's initiatives.</font></small></p>
+              </section>
 
-	  <!-- CONP -->
-	  <section data-background="images/CONP.png" data-background-size="100%">
+        <!-- TOSI -->
+	  <section>
+        <a href="https://www.mcgill.ca/newsroom/channels/news/mcgill-university-announces-transformative-20-million-donation-montreal-neurological-institute-and-264838"><img src="images/TOSI.png"></a>
 	  </section>
 
-	  <!-- TOSI -->
-	  <section>	
-	    <img src="images/TOSI.png">
-	  </section> 
-
-	  <!-- MNI Open Science CyberInfrastructure -->
-	  <section>		
-	    <h2><Demo</h2>
-		   <img src="images/Open_science_paper.png">
+              <!-- MNI Open Science CyberInfrastructure -->
+	  <section>
+		   <a href="https://www.frontiersin.org/articles/10.3389/fninf.2016.00053/full"><img src="images/Open_science_paper.png" width="75%" height="75%"></a>
+          <p><small>MNI's LORIS and CBRAIN platforms (more on these later) have been tasked with the technical challenges specific to the institutional-level implementation of open data sharing.</small></p>
```

Comment:
@christinerogers suggestions on improved wording for this blurb?

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/3/2